### PR TITLE
nolo tlc ciel 13_4_2

### PR DESCRIPTION
**Pull request by nolo16**
sorry for delay, there was a massive thunderstorm during the eclipse that fried like half of the appliances in my room (serious)

### CODE REVIEW

File: script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt [hunk 1]
```diff
@@ -1651,7 +1653,7 @@ For now, we should head toward the Church, right?"
 [sha:7941cb97428ad2441de0d19676798b8bc64136fd]{
 -- Page 496, Offset 34713. Mods: @k, @e.
 -- 「ええ。本国は無理でも、この国にはいくつかの支部がありますから。
-"Yes. The home country may be out of the question, but there are a few branches within this country.
+"Yes. The home country may be out of the question, but there are a few branches within Japan.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt:1656]: 2 things:
1. "home country" should probably just be "The Vatican" or something, leave a comment for the lore folk
2. my bilingual dictionary defines 本国 as "suzerain country" which is a really funny coincidence

File: script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt [hunk 2]
```diff
@@ -1662,37 +1664,37 @@ For now, we should head toward the Church, right?"
...
 -- Page 498, Offset 34717.
 -- 「……いえ、何でもありません。
-"...No, it's nothing to be concerned about.
+"...No, don't worry about it.
 }
 [sha:537748ccf6eb0d67accc57e6d8b25b32a095d473]{
 -- Page 498, Offset 34718.
 -- 今はここから出ましょう。わたしのアパートなら、少しは遠野くんを楽にしてあげられると思います」
-We should get out of here now. I think you'll be a little bit at ease if we go to my apartment."
+We should get out of here. I think you'll feel a little bit better if we go to my apartment."
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt:1682]: i know where this is going! I've read arc route!

**Review by Avuxo — APPROVED** [review]
otsukare
so are they going back to ciel's apartment or not...

File: script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt (hunk 2 above)
Comment (nolo16) [reply script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt:1682]: unfortunately not...............

**Pull request merged**